### PR TITLE
wgengine/router: set up basic IPv6 routing/firewalling.

### PR DESCRIPTION
**Pull request by danderson**
Part of #19.

Signed-off-by: David Anderson <danderson@tailscale.com>

---
This change is [Reviewable](https://reviewable.io/reviews/tailscale/tailscale/773)

### CODE REVIEW

**Comment by danderson** [comment]
Open question on this change is how IPv6 haters can break things by doing various degrees of IPv6 disablement. Same question re: older linuxes and busybox. I'm spinning up some VMs to test some of this now.

File: wgengine/router/router_linux.go [hunk 1]
```diff
@@ -90,6 +89,7 @@ type linuxRouter struct {
 	dns *dns.Manager
 
 	ipt4 netfilterRunner
```
Comment (bradfitz) [inline wgengine/router/router_linux.go:91]: look at that foresight!

**Review by bradfitz — APPROVED** [review]

**Comment by danderson** [comment]
@bradfitz PTAL again. Added a second commit to work around various broken IPv6 configs and old kernels. Not great, not terrible. But at least the router comes up on those older test boxes now.

**Comment by bradfitz** [comment]
Can look in a couple hours.

**Review by bradfitz — APPROVED** [review]

**Pull request merged**